db/schema.rb

```diff
@@ -45,8 +45,6 @@
     t.datetime "updated_at",     null: false
   end
 
-  add_index "games", ["organizer_id"], name: "index_games_on_organizer_id", using: :btree
```

Comment:
Lydia, please undo the changes to the schema.rb file, the schema should not be changed.